app.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   // return appConfig.metadataCacheUri
   metadataCacheUri:
     process.env.NEXT_PUBLIC_METADATACACHE_URI ||
-    'https://v4.aquarius.oceanprotocol.com',
+    'https://aquariusv4.oceanprotocol.com',
```

Comment:
why are we changing this URL again? `v4.aquarius.oceanprotocol.com` is the final URL and it's up, see #1004